### ISSUE TITLE
Could appium get IOS UIAccessibilityTraits value

### ISSUE THREAD

**Issue by maggie5750** [appium/appium#13630]
## The problem

We have some ios element has UIAccessibilityTraits which we need, but appium seems cannot get it. Do we have any method to get it or could you help to implement it?

## Environment

* Appium version (or git revision) that exhibits the issue: appium 1.15.1
* Last Appium version that did not exhibit the issue (if applicable): none
* Desktop OS/version used to run Appium: Mac 10.14.4
* Node.js version (unless using Appium.app|exe): 10.13.0
* Npm or Yarn package manager:
* Mobile platform/version under test:
* Real device or emulator/simulator:
* Appium CLI or Appium.app|exe:

## Details
We have some ios element has UIAccessibilityTraits which we need, but appium seems cannot get it. Do we have any method to get it or could you help to implement it?

**Comment by mykola-mokhnach** [comment]
I don't know any public XCTest methods, which would allow to do that. Perhaps, one could find something in the set of private XCTest APIs, although it is always a long trial, since there are, obviously, no documentation.
